ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/CaseData.java

```diff
@@ -169,4 +178,128 @@
+        hint = "Is the respondent’s current name different to their married name or the name shown on their marriage certificate?"
+    )
+    private YesOrNo d8PetitionerNameDifferentToMarriageCert;
+
+    @JsonProperty("D8PetitionerNameChangedHow")
+    @CCD(
+        label = "How did the petitioner change their name?"
+    )
+    private ChangedNameHow d8PetitionerNameChangedHow;
+
+    @JsonProperty("D8PetitionerNameChangedHowOtherDetails")
+    @CCD(
+        label = "Provide details of how they changed their name",
+        type = TextArea
+    )
+    private String d8PetitionerNameChangedHowOtherDetails;
+
+    @JsonProperty("D8DivorceWho")
+    @CCD(
+        label = "Who is petitioner divorcing?",
+        hint = "Husband or Wife?",
+        type = FixedList,
+        typeParameter = "WhoDivorcing"
+    )
+    private WhoDivorcing d8DivorceWho;
+
+    @JsonProperty("D8DerivedPetitionerHomeAddress")
+    @CCD(
+        label = "The Petitioner's home address",
+        type = TextArea
+    )
+    private String d8DerivedPetitionerHomeAddress;
+
+    @JsonProperty("D8PetitionerPhoneNumber")
+    @CCD(
+        label = "Petitioner's phone number",
+        regex = "^[0-9 +().-]{9,}$"
+    )
+    private String d8PetitionerPhoneNumber;
+
+    @JsonProperty("D8PetitionerContactDetailsConfidential")
+    @CCD(
+        label = "Keep the petitioner's contact details private from the respondent?",
+        type = FixedList,
+        typeParameter = "ConfidentialAddress"
+    )
+    private ConfidentialAddress d8PetitionerContactDetailsConfidential;
+
+    @JsonProperty("D8RespondentFirstName")
+    @CCD(
+        label = "Respondent's First name(s)",
+        hint = "Include all middle names here"
+    )
+    private String d8RespondentFirstName;
+
+    @JsonProperty("D8RespondentLastName")
+    @CCD(
+        label = "Respondent's Last name"
+    )
+    private String d8RespondentLastName;
+
+    @JsonProperty("D8RespondentNameAsOnMarriageCertificate")
+    @CCD(
+        label = "Name changed since marriage?",
+        hint = "Is the respondent’s current name different to their married name or the name shown on their marriage certificate?"
+    )
+    private YesOrNo d8RespondentNameAsOnMarriageCertificate;
+
+    @JsonProperty("RespNameDifferentToMarriageCertExplain")
+    @CCD(
+        label = "Please explain, if known, how their name has changed since they were married.",
+        type = TextArea
+    )
+    private String respNameDifferentToMarriageCertExplain;
+
+    @JsonProperty("PetitionerSolicitorName")
+    @CCD(
+        label = "Petitioner Solicitor’s name"
+    )
+    private String petitionerSolicitorName;
+
+    @JsonProperty("D8SolicitorReference")
+    @CCD(
+        label = "Your reference number"
+    )
+    private String d8SolicitorReference;
+
+    @JsonProperty("PetitionerSolicitorPhone")
+    @CCD(
+        label = "Petitioner Solicitor Phone number",
+        regex = "^[0-9 +().-]{9,}$"
```

Comment:
should we have the phone number validation in ccd ?
In case of save & signout if user enters incorrect phone number front end won't be able to save case ?

@linusnorton should we have validation on front end instead ?

Reply:
I suppose the issue here is that the regex here controls the validation of the UI for the Solicitor journey.  But we don't want it to apply when submitting from the Citzen.

Reply:
yes that's correct as we use same case data model it will apply to citizen and solicitor

Reply:
isn't there a type of Email?